### PR TITLE
FIX Fix link plugins to support tinymce6

### PR DESCRIPTION
**Pull request by GuySartorelli**
Updates the anchor and page on this site tinymce plugins for tinymce6 support

## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10369

### CODE REVIEW

**Comment by GuySartorelli** [comment]
The failed tests pass locally - when https://github.com/silverstripe/silverstripe-admin/pull/1367 has been merged these should go green.

**Review by sabina-talipova — APPROVED** [review]

**Comment by sabina-talipova** [comment]
The branch will be merged after merging of [silverstripe-admin](https://github.com/silverstripe/silverstripe-admin/pull/1367) new changes.

**Comment by GuySartorelli** [comment]
Fixed behat tests - these are in line with other behat tests that use the tinymce wysiwyg controls.
Note that https://github.com/silverstripe/silverstripe-asset-admin/pull/1295 may need to be merged for this to go green.

File: tests/behat/src/FixtureContext.php [hunk 1]
```diff
@@ -118,22 +118,6 @@ public function iSeeTheRadioButtonAttributeEquals($radioLabel, $attribute, $valu
...
     }
 
-    /**
-     * Assumes you've just opened the Insert link menu, e.g.
-     * I click on the "div[aria-label='Insert link [Ctrl+K]'] button" element
-     *
-     * @When /^I select "(.+?)" from the TinyMCE menu with javascript$/
-     * @param string $label
-     */
-    public function iSelectFromTheTinyMceMenu($label)
```
Comment (ChloeSartorelli) [inline tests/behat/src/FixtureContext.php:128]: This is no longer used - it was only used in the above tests where it is no longer necessary.

**Comment by GuySartorelli** [comment]
Every time I fix an issue, another one appears lol. This time the behat failures were because if you select some text and then click to add a link, it wasn't adding a link based on the existing content you had selected.
https://github.com/silverstripe/silverstripe-admin/pull/1372 fixes that and will need to be merged for this to go green.
Fingers crossed that's the last of it.

**Pull request merged**